### PR TITLE
Translate words in activations.md

### PR DESCRIPTION
**Pull request by JoeyValentine**
#80 
단어 수정 완료했습니다!

### CODE REVIEW

File: sources/activations.md [hunk 1]
```diff
@@ -34,19 +35,19 @@ keras.activations.softmax(x, axis=-1)
...
-- __axis__: 정수, softmax normalization이 적용되는 axis.
+- __x__: 입력 텐서.
+- __axis__: `int`. Softmax 정규화<sub>normalization</sub>가 적용되는 축<sub>axis</sub>.
 
 
-__Returns__
+__반환값__
 
-Softmax 변환으로 생성된 tensor.
+Softmax 변환으로 생성된 텐서.
```
Comment (fuzzythecat) [inline sources/activations.md]: sigmoid, softplus 처럼 반환값에 수식을 포함하게 바꾸면 좋을 것 같습니다

Comment (JoeyValentine) [reply sources/activations.md]:
```python
e = K.exp(x - K.max(x, axis=axis, keepdims=True))
s = K.sum(e, axis=axis, keepdims=True)
return e / s
```
element-wise 연산이므로 exp(x) / sum(exp(x)) 로 표현하는게 어떨까요?

File: sources/activations.md [hunk 2]
```diff
@@ -160,25 +161,25 @@ keras.activations.relu(x, alpha=0.0, max_value=None, threshold=0.0)
...
+- __x__: 입력 텐서.
+- __alpha__: `float`. 음수 부분의 기울기. 디폴트 값은 0.
+- __max_value__: `float`. 포화 임계값.
+- __threshold__: `float`. 활성화를 위한 임계치.
 
-__Returns__
+__반환값__
 
-A tensor.
+ReLU 변환으로 생성된 텐서.
```
[the remark this answers is not in the document]
Comment (JoeyValentine) [reply sources/activations.md]: 함수 설명부분에 반환값 형식을 설명하고 있는데, 
이 부분을 삭제하고 함수 반환값 부분에 
```
x >= max_value 일 때 f(x) = max_value,
threshold <= x < max_value 일 때 f(x) = x,
그 외는 f(x) = alpha * (x - threshold).
```
를 작성하는게 어떨까요?

**Comment by JoeyValentine** [comment]
수정 완료했습니다!

**Comment by fuzzythecat** [comment]
감사합니다!
